src/yuzu/bootmanager.cpp

```diff
@@ -319,10 +320,18 @@ GRenderWindow::~GRenderWindow() {
 
 void GRenderWindow::OnFrameDisplayed() {
     input_subsystem->GetTas()->UpdateThread();
+    TasInput::TasState new_tas_state = std::get<0>(input_subsystem->GetTas()->GetStatus());
```

Comment:
```suggestion
    const TasInput::TasState new_tas_state = std::get<0>(input_subsystem->GetTas()->GetStatus());
```

Since we try to make mutability explicit if possible